### ISSUE TITLE
Policy on archaic terms and senses

### ISSUE THREAD

**Issue by Marcusjmdict** [JMdictProject/JMdictIssues#140]
We don't, to my knowledge, have a clear policy on when we need to include [arch] entries and senses. Historically, we tended to include everything. Recently, not as much.

I suggest we enshrine in policy that
1) [arch] entries should only be included unless there's a good reason to have them. I can't come up with any criteria other than that they should be relatively well-known (through use in a famous poem, costume dramas, etc.) to modern users of the language or perhaps frequently commented on in modern discourse. 
2) I feel the same about [arch] senses, and I don't think we should make excemptions for "original meaning" etc. (if necessary, we can use [lit] on the contemporary senses)

I also don't think we should have a "grandfather clause" but strive to get rid of perhaps most of the stuff we have tagged as [arch].

(I think all editors are aware but by "arch" we mean words that were exclusively used during the Edo period)

**Comment by ReneMalenfant** [comment]
I think the archaic senses are one of the strengths of the dictionary and that it would be a shame to lose them. I'd say they should be included whenever possible. (It's always possible for viewer software to have an option to hide archaic senses by default, etc.)

**Comment by Marcusjmdict** [comment]
I assumed most of us were on the same page so didn't really make an argument for the suggestions, apologies. But the argument is essentially "scope". Jmdict is a dictionary of contemporary Japanese. We have to draw the line somewhere. Arch entries and senses are useful only to a very small minority of people, and clutter to everybody else. The people who find this useful are certainly a minority within a minority, as most people who do read classical Japanese texts are likely happy with Japanese dictionaries. Allowing people to submit new arch entries and senses represent more work for the editors, who largely are not competent in classical Japanese anyway (speaking of myself, at minimum).

**Comment by ReneMalenfant** [comment]
I don't agree that JMdict is a dictionary of _contemporary_ Japanese. It's a _comprehensive_ dictionary:
https://www.edrdg.org/wiki/index.php/JMdict-EDICT_Dictionary_Project

> The JMdict/EDICT project has as its goal the production of a comprehensive freely-available Japanese/English Dictionary database in machine-readable form which can be used by a variety of applications and servers.

Obscure senses are just as much 'clutter' as archaic senses. I don't see why either would be removed, if they contain correct information. The inclusion of the archaic senses is one of the things that sets this dictionary apart and makes it more useful than Eijiro, GG5, etc.

The point of the dictionary is to be comprehensive, in my view. No one needs to be exposed to [arch] or [obsc] senses. Viewer software can remove them all from view.

**Comment by Marcusjmdict** [comment]
I guess the dictionary is what we want it to be, but the editorial policy does state that our goal is to reflect modern Japanese.
Edit: didn't mean to close the issue.

**Comment by JMdictProject** [comment]
I agree with Rene's comments. I think the dictionary should be comprehensive and should certainly not shun terms that date from the Edo period or earlier. The only condition I would place on the addition of such terms is that they have some presence in current sources, e.g. as indicated by WWW hits and/or n-grams, and are supported by kokugos (preferably not Nikkoku alone).

I certainly don't support culling existing entries or senses unless they are wrong.

**Comment by ReneMalenfant** [comment]
I agree with not using only Nikkoku as the only source for something archaic. That seems like a good cutoff for me.

**Comment by robinjmdict** [comment]
I agree with Marcus.

I don't think our current policy of accepting almost any archaic term that meets some low n-gram threshold is practical or helpful.

Although JMdict/EDICT has been around for over 30 years, our coverage of archaic terms/senses is extremely sparse. By my estimation, there are over 100,000 archaic terms and senses in the kokugos that we don't currently have in JMdict. It's probably fair to say that JMdict will never be comprehensive with respect to archaic words and senses. It would be a huge amount of work and the demand just isn't there.

We already ignore most archaic senses of words used in contemporary Japanese. For this reason alone, JMdict is not a useful resource for pre-modern Japanese.

Another issue is the accuracy of the glosses – something we cannot guarantee when none of the editors are native speakers or Japanese scholars and we only have dictionary definitions to work from.

There's also a time cost to our current policy. I've cumulatively spent many hours reviewing and editing submissions of archaic terms – time that I could have spent much more productively on other entries. Many (possibly the vast majority) of these terms were submitted by people who encountered the word in a kokugo dictionary. Most of them will probably never be looked up in JMdict.

My proposal is to stop including archaic terms (and eventually delete existing archaic terms) unless they meet at least one of the following criteria:

1. The term is listed in a JE or smaller kokugo dictionary.
2. The term appears in a well-known book, play, song, proverb, idiom, quote, etc.
3. There is considerable evidence of the term's use outside of old texts (e.g. a high n-gram count)

This would give us coverage of the archaic terms that people are likely to look up while also making the editors' job easier.

**Comment by ReneMalenfant** [comment]
It's an incremental project. So it seems perfectly reasonable and understandable to me that not all archaic terms are currently included and that they might never be. (The same goes for all words.)

If we don't feel comfortable approving a particular archaic sense of a word, then I don't think we should feel obligated to do so (or to spend hours as a volunteer figuring it out). If no one is comfortable vetting it, then it shouldn't make it into the dictionary. But if it can be vetted, then I think it should. That applies not only to archaic terms but to all sorts of terms that aren't already included in some other JE dictionary. (And we have tons and tons of glosses based on our own translations and not copied or verified using other JEs.) And the more archaic terms we include in the dictionary, the easier it will be to vet new ones, because translating the kokugos will be easier.

If you're creating a new entry for any word, I don't think you should feel obligated to translate every sense of the word (archaic or not). Capturing the most commonly used senses is clearly the most efficient use of time if you don't feel like really digging in to the obscure and archaic senses. And someone who is motivated to do so can always come along later and add the senses that you didn't include. It's an incremental project; I don't think we have to be completists out of the box. (Maybe some day, someone with a PhD in archaic Japanese will join the project and build on what's currently there. Who knows.)

I don't understand the motivation for removing correct information from an electronic dictionary where size is not an issue. Put everything in it and let software hide the things you don't want to see.

**Comment by parfait8566** [comment]
> Another issue is the accuracy of the glosses – something we cannot guarantee when none of the editors are native speakers or Japanese scholars and we only have dictionary definitions to work from.

But I don't think accuracy is ever guaranteed, even with modern Japanese entries. As someone who only occasionally contributes, I've already seen plenty of slips (spelling mistakes, translation errors, glosses missing the point or lacking the right nuance, incorrect tags, etc.) and I'm not saying this to disparage anybody, I really respect the work you guys do. I just think mistakes are in the nature of projects like this. 

> That applies not only to archaic terms but to all sorts of terms that aren't already included in some other JE dictionary. (And we have tons and tons of glosses based on our own translations and not copied or verified using other JEs.)

I agree with this. And with archaic entries you have the bonus of Japanese dictionaries almost always including direct quotes which can really help tune down the amount of possible inaccuracies.

I'll also say that I find archaic Japanese in any form (words, verbs, specific kanji forms, etc.) used very frequently in the modern literature I consume. It might be niche and I'm definitely not gonna argue most Japanese learners encounter it on a daily basis, but it still comes handy. So I'm strongly against the idea of removing archaic entries or even stop including them.

On another note, JMdict should add a new tag for the 文語 verb 来 (く) which is カ行変格活用, the existing [vk] only covers 来る.

**Comment by JMdictProject** [comment]
Just a quick comment on the dimensions of the "archaic" issue. We currently have a little over 3k entries with at least one sense tagged [arch].  From sampling it seems about 1k of those have the tag on a later sense from the first. Overall we're talking about around 1% of entries.

**Comment by Marcusjmdict** [comment]
> But I don't think accuracy is ever guaranteed, even with modern Japanese entries. As someone who only occasionally contributes, I've already seen plenty of slips (spelling mistakes, translation errors, glosses missing the point or lacking the right nuance, incorrect tags, etc.) and I'm not saying this to disparage anybody, I really respect the work you guys do. I just think mistakes are in the nature of projects like this.

I think this is completely besides the point Robin was making. All dictionaries have issues of the sort you've mentioned, but it is one thing for a group of individuals more or less proficient in English and Japanese to compile a Japanese-English dictionary based on their understanding of both languages with the aid of Japanese sources, and another thing entirely for them to rely entirely on Japanese defitinions alone of another language (which I think we can agree classical Japanese is). Using Japanese defiitions of Somali words we could perhaps create Somali-English dictionary, but why should anybody want to use it other than out of desperation?

I don't really think it's our place to attempt this. I think it needlessly lowers the quality of the dictionary for quantity. I agree with Robin's 3 criteria.

**Comment by ReneMalenfant** [comment]
I'm not sure how including archaic terms 'lowers the quality of the dictionary'.

Even if I were to grant that the archaic entries are lower quality on average than the non-archaic entries, the dictionary is still of higher quality for including them, as long as they're not wrong (i.e., negative quality) on average. And I'm pretty sure that's not the case.

If you like Coke a lot and Pepsi just a little, and your fridge contains 12 Cokes and 2 Pepsis, you don't throw the Pepsi in the trash just to increase the average quality of soda in your fridge. Your fridge contains more things you like to drink if you keep the Pepsis too.

**Comment by Marcusjmdict** [comment]
I don't think it should be a contentious argument - whatever you want to grade our contemporary Japanese-English part in terms of quality, the archaic part would absolutely be graded lower. It follows, then, that the average quality of both parts put together is  lower than what the contemporary part would be taken on its own. Why shouldn't we or our users assess them together? 
 I understand your argument as "quantity is actually quality" but I disagree.

**Comment by ReneMalenfant** [comment]
So, in your view, an ideal dictionary is one that contains one extremely well-defined word?

I'm being cheeky, but I think comprehensiveness is quite obviously part of the quality of a dictionary (especially one that explicitly aims to be comprehensive). And it's a measure of quality that is obviously not captured by the average quality of glosses.

**Comment by Marcusjmdict** [comment]
That's quite a compelling idea. I could imagine a moderately interesting short story written on the topic. 

Seriously speaking, classical Japanese is not the same language as modern Japanese. Should we include all Okinawan words in the dictionary? All ainu words? OK - why not then also Korean? I don't think it's outlandish to try and keep the scope of the dictionary to the two languages all editors are relatively competent in.

**Comment by parfait8566** [comment]
> All dictionaries have issues of the sort you've mentioned, but it is one thing for a group of individuals more or less proficient in English and Japanese to compile a Japanese-English dictionary based on their understanding of both languages with the aid of Japanese sources, and another thing entirely for them to rely entirely on Japanese defitinions alone of another language (which I think we can agree classical Japanese is).

I don't think we can agree on that, at the very minimum not to the extent you're implying. Putting classical Japanese in the same box as Somali (or Chinese or Korean, for that matter) in terms of linguistic distance to modern Japanese sounds very perplexing to me.
Classical Japanese is significantly intelligible by modern speakers (without necessarily counting the fact it's still studied in high school), continues to appear frequently in modern literature and is prevalent in proverbs, idioms and so on. Even if the Somali example was hyperbolic, it's still a pretty disingenuous comparison. There's a reason Daijirin has classical Japanese entries and not Korean entries.
I also think the idea that modern entries are necessarily of higher quality than classical entries is unfounded. What about adding translations for words none of the editors encountered before? Or new slang that is not backed up by many Japanese sources? Compare that to classical entries in Japanese dictionaries almost always having direct quotes from books (which in turn often happen to have modern Japanese translations). Many times, the definition is quite unambiguous. Or for example all sources agree that one specific reading is archaic. There's nothing "low quality" about this.
